docs/content/v2.6/benchmark/key-value-workload-ycql.md

```diff
@@ -64,7 +64,7 @@ We will use the [YugabyteDB Workload Generator](https://github.com/yugabyte/yb-s
 To get the tool (``yb-sample-apps.jar`), run the following command.
 
 ```sh
-$ wget https://github.com/yugabyte/yb-sample-apps/releases/download/1.3.9/yb-sample-apps.jar?raw=true -O yb-sample-apps.jar
+$ wget 'https://github.com/yugabyte/yb-sample-apps/releases/download/1.3.9/yb-sample-apps.jar?raw=true' -O yb-sample-apps.jar
```

Comment:
We no longer support 2.6 nor its docs

Reply:
(on the other hand, might as well make this change here, since it'll be helpful)